modules/experimental-layers/src/path-outline-layer/path-outline-layer.js

```diff
@@ -2,6 +2,7 @@ import {PathLayer} from '@deck.gl/layers';
 import GL from '@luma.gl/constants';
 import {Framebuffer, Texture2D} from 'luma.gl';
 import outline from '../shaderlib/outline/outline';
+import assert from '../../../core/src/utils/assert';
```

Comment:
You cannot import something from another module. This is not going to be the directory structure when the module is published.